wagtail/admin/templates/wagtailadmin/generic/base.html

```diff
@@ -7,9 +8,34 @@
     {% block header %}
         {% block slim_header %}
             {% if breadcrumbs_items %}
+                {% fragment stripped as actions %}
```

Comment:
Use the `stripped` option so that if both blocks are empty, the resulting fragment is actually empty instead of being a string consisting of only whitespaces. This ensures that when the fragment is passed down to the `slim_header.html`, you can check its emptiness with `if`

(horrible, I know, but give me something better?)

Reply:
Indeed pretty horrible! This doesn’t strike me as a dealbreaker but definitely one for a list of refactorings.